control-plane-operator/controllers/hostedcontrolplane/oapi/deployment.go

```diff
@@ -83,7 +82,24 @@ func openShiftAPIServerLabels() map[string]string {
 	}
 }
 
-func ReconcileDeployment(deployment *appsv1.Deployment, auditWebhookRef *corev1.LocalObjectReference, ownerRef config.OwnerRef, config *corev1.ConfigMap, auditConfig *corev1.ConfigMap, serviceServingCA *corev1.ConfigMap, deploymentConfig config.DeploymentConfig, image string, socks5ProxyImage string, etcdURL string, availabilityProberImage string, internalOAuthDisable bool, platformType hyperv1.PlatformType, hcpAdditionalTrustBundle *corev1.LocalObjectReference, clusterConf *hyperv1.ClusterConfiguration) error {
+func ReconcileDeployment(deployment *appsv1.Deployment,
+	auditWebhookRef *corev1.LocalObjectReference,
+	ownerRef config.OwnerRef,
+	config *corev1.ConfigMap,
+	auditConfig *corev1.ConfigMap,
+	serviceServingCA *corev1.ConfigMap,
+	deploymentConfig config.DeploymentConfig,
+	image string,
+	konnectivityProxyImage string,
```

Comment:
It's not obvious to me without diving into the code what konnectivityProxyImage refers here? Is it https://github.com/kubernetes-sigs/apiserver-network-proxy or is it our binary? I think it should be either konnectivityServerImage or konnectivityHTTPProxyImage?